modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,120 @@
+. Investigate each additional network interface to find relevant pod annotations. These annotations differ depending on the kind of network interface you have: 
+.. For SR-IOV, look for the annotated fields:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: openshift.io/netpolicy67619
+  creationTimestamp: "2024-10-03T20:40:48Z"
+  generation: 1
+  name: netdevice67619-1
+  namespace: openshift-sriov-network-operator
+  resourceVersion: "159492"
+  uid: 13131f06-e7b1-4b42-8d95-ac7c53e908cb
+spec:
+  config: |-
+    {
+        "cniVersion": "1.0.0",
+        "name": "netdevice67619-1", <1>
+        "type": "sriov",
+        "vlan": 0,
+        "vlanQoS": 0,
+        "logLevel": "info",
+        "ipam": {
+            "type": "static",
+            "addresses": [
+                {
+                    "address": "192.168.122.71/24" <2>
+                }
+            ]
+        }
+----
+<1> Name of the virtual virtual machine connected to the additional network interface.
+<2> The MAC address to identify the virtual machine.
+
+.. For L2, look for the annotated fields:
++
+[source,yaml]
+----
+ # ...
+  {
+    "name": "ovn-kubernetes",
+    "interface": "eth0",
+    "ips": [
+      "10.129.2.39"
+    ],
+    "mac": "0a:58:0a:81:02:27",
+    "default": true,
+    "dns": {}
+  },
+  {
+    "name": "my-vms/l2-network",   <1>
+    "interface": "podc0f69e19ba2",
+    "mac": "02:fb:f8:00:00:12",    <2>
+    "dns": {}
+  }
+ # ... 
+----
+<1> The name of the virtual machine connected to the additional network interface.
+<2> The MAC address to identify the virtual machine.
+
+. Configure `FlowCollector` based on the information you found from the additional network investigation. 
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+# ...
+ebpf:
+    privileged: true       <1>
+  processor:
+    advanced:
+      secondaryNetworks:
+      - index:
+        - MAC "02:fb:f8:00:00:12"  <2>
+        - IP  "192.168.122.71/24"  <3>
+        name: my-vms/l2-network    <4>
+# ...
+----
+<1> Ensure that the ebpf agent is in `privileged` mode so that the flows are enriched according to the MAC address.
+<2> If your additional network information has a MAC address, specify add it here.
+<3> If your additional network information has an IP address, specify add it here.
+<4> Specify the name of the virtual machine. 
+
+. Ping from one VM to another for secondary interface IP
```

Comment:
```suggestion
[source,yaml]
----
apiVersion: flows.netobserv.io/v1beta2
kind: FlowCollector
metadata:
  name: cluster
spec:
# ...
ebpf:
    privileged: true       <1>
  processor:
    advanced:
      secondaryNetworks:
      - index: <2>
        - Interface
        - MAC
        - IP
        name: my-vms/l2-network    <3>
# ...
----
<1> Ensure that the ebpf agent is in `privileged` mode so that the flows are enriched according to the MAC address.
<2> Define the fields to use for indexing the virtual machine pods. These should form a unique Pod identifier across the cluster.
<3> Specify the name of the network. Usually <namespace>/<network-attachement-definition-name>. 
```

Reply:
Discussed this with @skrthomas on slack -  that we should keep general recommendation such as indexing by MAC only. For other fields we could simply add a note saying, indexing is also supported for IP and Interface fields in combination with MAC if they’re published in annotations.